redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewFlexContainer.kt

```diff
@@ -37,16 +39,31 @@ internal class ViewFlexContainer(context: Context, direction: FlexDirection) {
     flexDirection = direction
   }
 
-  private val _view = HostView(context)
-  val view: View get() = _view
+  private val hostView = HostView(context)
+
+  private val scrollView = if (direction.isHorizontal) {
+    HorizontalScrollView(context).apply {
+      isFillViewport = true
+      setTouchEnabled(false)
+      addView(hostView)
+    }
+  } else {
+    VerticalScrollView(context).apply {
+      isFillViewport = true
```

Comment:
`HorizontalScrollView` and `VerticalScrollView`'s common parent is `FrameLayout`, which doesn't define `isFillViewport` so we need to set it on each view.